src/dune_rules/coq/coq_lib.ml

```diff
@@ -153,6 +153,7 @@ and Legacy : sig
     ; installed_root : Path.t (* ; libraries : (Loc.t * Lib.t) list Resolve.t *)
     ; vo : Path.t list
     ; cmxs : Path.t list
+    ; cmxs_directories : Path.t list
```

Comment:
can you add a comment distinguishing this from the field above. It seems strange that we have both the cmxs files themselves and the directories we find them in.

Reply:
In fact `cmxs` files are not used anywhere, oh, actually I think they should be in as coqdep requires re-run when the list of cmxs files change! (Not the contents) Yuck!

Reply:
I removed the cmxs field from coq_lib as it is not used, still we keep it in the scanning code for debug purposes.